src/objects/ecatt/zcl_abapgit_ecatt_script_downl.clas.abap

```diff
@@ -53,7 +53,7 @@ CLASS zcl_abapgit_ecatt_script_downl DEFINITION
         RAISING
           cx_ecatt_apl_util.
 
-ENDCLASS.
+  ENDCLASS.
```

Comment:
Bad indentation in this line, the CLASS and ENDCLASS should start at the same column